website/sidebars.json

```diff
@@ -1,52 +1,49 @@
 {
-    "guide": [
-        "guide/welcome",
+  "guide": [
+    "guide/welcome",
+    {
```

Comment:
Looks like a lot of spacing issue here. Please match with the existing one.

Reply:
Fixed